pallets/loans/src/functions.rs

```diff
@@ -187,23 +183,20 @@ impl<T: Config> Pallet<T> {
 		let count = active_loans.len();
 		ActiveLoans::<T>::insert(pool_id, active_loans);
 
-		Ok(count.try_into().unwrap())
+		Ok((count.try_into().unwrap(), interest_rate_per_sec))
```

Comment:
This will be a silent error in case we allow above `u32::max errors`. Probably never going to happen but I don't like unwrapping in the runtime.

Reply:
Changed this to an `expect` explaining why it can't fail. Unfortunately these conversions between `usize` and `u32` are _everywhere_ due to Parity using u32 in lots of APIs that then map to a Rust `usize`